README.md

```diff
@@ -308,7 +308,7 @@ You have encountered an issue and wish to report it to the VLC dev team?
 
 You can create one on our [Gitlab](https://code.videolan.org/videolan/LibVLCSharp/issues) (using an issue template) for .NET related questions/problems.
 
-_For LibVLC/VLC specific issues, usually native code, trac is our [bug tracker](https://trac.videolan.org/vlc/)._
+_For LibVLC/VLC specific issues, usually native code, VLC GitLab is our [bug tracker](https://code.videolan.org/videolan/vlc/-/issues)._
```

Comment:
I'd rephrase it as GitLab is not a bug tracker per se. What about "native code, please go to the [bug tracker of the VLC project]"?

Reply:
Sounds good! I will change it.